src/utils/Chart.js

```diff
@@ -0,0 +1,42 @@
+const d3 = require('d3');
+
+const Chart = {
+  getDataMinMaxValues (data, axis) {
+    let max = d3.max(data, d => {
+      return d[axis];
+    });
+
+    let min = d3.min(data, d => {
+      return d[axis];
+    });
+
+    const maxDigits = Math.ceil(max).toString().length - 1;
+    const multiplier = Math.pow(10, maxDigits);
+
+    max = Math.ceil(max / multiplier) * multiplier;
+    min = Math.floor(min / multiplier) * multiplier;
+
+    return { min, max };
+  },
```

Comment:
@tumentumurchudur @jmophoto Ok I revamped this a little.  we now figure out the correct multiplier based upon the number of digits in the max value after it has been rounded to a whole number(int).  We then use the `Math.pow` function that gets the correct multiplier to use.  Tested with several data sets with small and large numbers and everything is working correctly.